docs/src/introduction.md

```diff
@@ -3,33 +3,32 @@
 > *This documentation is still Work In Progress. Some topics have been discussed in greater depth, while others require additional clarification. Sections of this documentation might later be reorganized in order to achieve a better flow.*
 
 ## Welcome to the Polygon Miden Documentation
-Polygon Miden is a zkRollup for high-throughput and private applications. Anyone can create zero-knowledge proofs of their own state changes, and Miden only verifies. It is expected to launch a public testnet in Q4.
+Polygon Miden is a zkRollup for high-throughput and private applications. Users can create zero-knowledge proofs of their own state changes, and the Miden network verifies. It is expected to launch a public testnet in Q1 2024.
 
-This documentation explains the Miden architecture and concepts, the network and its components, and the underlying cryptographic primitives.
+This documentation explains how to use Miden. Furthermore, it explains the Miden architecture and concepts, the network and its components, and the underlying cryptographic primitives.
 
 Press `s` or for full-text search in the docs.
```

Comment:
Not a part of this PR, but I'm not sure if we need this line here (also, if we migrate docs to some other tool, this may not be relevant).